vanilla-framework/vanilla-framework
# Fixed the known issues with the vertical spacing

## Done
Fixed the known issues with the vertical spacing. Added more template tests.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/vertical-spacing
- Check that all the spacing is good

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1338
